client/src/pages/RegisterPage/index.tsx

```diff
@@ -43,19 +33,18 @@ export const RegisterPage = () => {
       });
   };
 
-  const handlePrivacyCheck = useCallback(() => {}, []);
   const handleClickRegisterButton = useCallback(async (e: React.MouseEvent<HTMLElement>) => {
     e.preventDefault();
     await sendInfoToServer();
   }, []);
 
   useEffect(() => {
-    if (id && interest.length > 0 && techStack.length > 0 && privacyCheck) {
+    if (id && interest.length > 0 && techStack.length > 0) {
       setIsAllSet(true);
     } else {
       setIsAllSet(false);
     }
```

Comment:
```suggestion
      setIsAllSet(id && interest.length > 0 && techStack.length > 0);
```
조건문 없이 한 문장으로 해결될것같습니다

Reply:
깔끔한 리팩토링 감사합니다 :)
아래와 같이 수정해두었습니다.
```
  useEffect(() => {
    setIsAllSet(id.length > 0 && interest.length > 0 && techStack.length > 0);
  }, [id, interest, techStack]);